recipes/secimtools/meta.yaml

```
@@ -1,14 +1,14 @@
 {% set name = "secimtools" %}
-{% set version = "21.3.4.2" %}
+{% set version = "21.6.3" %}
 {% set hash_type = "sha256" %}
-{% set hash_value = "ce8f8fd5f63f5d31f3a5c63350caa4f5bda7e660081a697585a5ff41abdbbba1" %}
+{% set hash_value = "e6a1946e505d80f1d39c18989e9d877b9ad3867d4e393e1ca4562632589c6f4e" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
-  url: https://files.pythonhosted.org/packages/0a/b7/20d14d2fd036494cf4711d0fbb1bd1add00aca4b6ca49fb841f3e7d61ff5/{{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/a5/c3/01b2c4d97efd1482ca562fd1a589b2f3ff00a2948de1ce63b752d3bbc922/{{ name }}-{{ version }}.tar.gz
```

This may make it so that BiocondaBot can automatically update your package for you:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```